src/any_llm/providers/anthropic/utils.py

```
@@ -41,6 +56,14 @@
 }
 
 
+def _refusal_stop_details(value: Any) -> dict[str, Any] | None:
```

Ruff (0.16.1)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)